modules/gapi/perf/common/gapi_core_perf_tests.hpp

```diff
@@ -70,7 +70,7 @@ namespace opencv_test
     class ConcatVertPerfTest : public TestPerfParams<tuple<cv::Size, MatType, cv::GCompileArgs>> {};
     class ConcatVertVecPerfTest : public TestPerfParams<tuple<cv::Size, MatType, cv::GCompileArgs>> {};
     class LUTPerfTest : public TestPerfParams<tuple<MatType, MatType, cv::Size, cv::GCompileArgs>> {};
-    class ConvertToPerfTest : public TestPerfParams<tuple<MatType, int, cv::Size, cv::GCompileArgs>> {};
+    class ConvertToPerfTest : public TestPerfParams<tuple<compare_f, MatType, int, cv::Size, double, double, cv::GCompileArgs>> {};
```

Comment:
Does the performance really depend on these two doubles?

I believe it was not there as their contribution to the kernel's performance is constant regardless of the value.

Reply:
There is the difference: OCV implementation calls an OpenCV function which works differently (and much slower, up to x3) if `alpha` and `beta` are not default (`1` and `0` respectively); so I decided it is important to compare these cases.
But  you are totally right, performance doesn't depend on what is the non-default value, so there is no need in so many tests.
Will reduce to 2 different cases (instead of 9)

Reply:
Got it, makes sense, thanks!